docs/hints-for-usage.md

```diff
@@ -0,0 +1,17 @@
+# Hints for JOPA Usage
+This page will list useful tips/hints that might help developers implement a JOPA-based solution.
+
+## Making Inferred Attribute Writable 
+
+### Problem
+In JOPA, inferred attributes are not writable. However, sometimes one wants to both modify the property and fetch its values that are inferred.
+
+An example would be the property skos:exactMatch. It might be both edited by an application (e.g. a SKOS editor) and inferred (e.g. because it is symmetric).
+
+### Workaround
+![image](https://user-images.githubusercontent.com/1140626/115027430-fddf1900-9ec3-11eb-90db-f5541bf7bd45.png)
```

Comment:
within picture you have mistake -- exactMatchInferred instead of exactMatchAsserted within JOPA anotation

+ directed edge would be more clear within the picture